jem.js

```diff
@@ -202,10 +204,20 @@ if (typeof Inaka.Jem == 'undefined') Inaka.Jem = {};
      return [dv, i + k];
    }
 
-   function _decodeString(dv, i)
+   function _decodeString(dv, i, lenBytes = 4)
```

Comment:
@sildur I know it's ES6 only, do we care? I can let it default to ``undefined`` I guess. Otherwise, how do I remove the warning?